docs/communication/event-emitter/event-emitter.md

```diff
@@ -1,6 +1,6 @@
 ### Event-emitter pattern
 
-So far we've just seen how to communicate components passing down data from parent to children. And how about the other way around ?.
+So far we've just seen how to communicate components passing down data from parent to children. And how about the other way around?
```

Comment:
Again, let's try to standardize this part of the redaction, item of every list should end up with dot.